source/common/upstream/cluster_manager_impl.cc

```diff
@@ -182,6 +182,9 @@ ClusterManagerImpl::ClusterManagerImpl(const Json::Object& config, ClusterManage
 
   // To avoid threading issues, for those clusters that start with hosts already in them (like
   // the static cluster), we need to post an update onto each thread to notify them of the update.
+  // We also require this for dynamic clusters where an immediate resolve
```

Comment:
nit: Can you flow this comment out to 100 or reflow the entire thing to 80. It just looks strange.